discovery/srv.go

```diff
@@ -68,8 +69,8 @@ func SRVGetCluster(name, dns string, defaultToken string, apurls types.URLs) (st
 				n = fmt.Sprintf("%d", tempName)
 				tempName += 1
 			}
-			stringParts = append(stringParts, fmt.Sprintf("%s=%s%s", n, prefix, tcpAddr.String()))
-			log.Printf("discovery: Got bootstrap from DNS for %s at host %s to %s%s", service, host, prefix, tcpAddr.String())
+			stringParts = append(stringParts, fmt.Sprintf("%s=%s%s", n, prefix, host))
+			log.Printf("discovery: Got bootstrap from DNS for %s at %s%s", service, prefix, host)
```

Comment:
After some researching, Chrome drops trailing dot at hostname verification (`X509Certificate::VerifyHostname`). So we should do drop at same timing (verification)... hmm this may be golang's issue.

what should be better workaround? dropping trailing dot at this function?